### PR TITLE
Vaurca Drink Additions

### PR DESCRIPTION
**Pull request by senorsapo**
- rscadd: "Adds Sedantis Water to the Zo'ra Soda vendor."
  - rscadd: "Adds Skye'mok in a V'krexi organ to the Booze-o-Mat"
  - rscdel: "Adds a new sprite for the Flagsdale Mule drink."

Basically, you get vendored the organ in a wrapper, you open the wrapper and pull the organ (which is in-code the bottle) out.

### CODE REVIEW

**Comment by senorsapo** [comment]
!review

**Comment by desvenlafaxine** [comment]
Lore approved.

File: code/game/objects/items/vaurca.dm [hunk 1]
```diff
@@ -134,3 +134,34 @@
...
+	. = ..()
+	make_exact_fit()
+
+/obj/item/storage/box/fancy/vkrexi_swollen_organ/attack_self(mob/user)
+	open(user)
+
+/obj/item/storage/box/fancy/vkrexi_swollen_organ/update_icon()
+	. = ..()
+	if(opened)
+		if(contents.len == 0)
```
Comment (FluffyGhoster) [inline code/game/objects/items/vaurca.dm:164]:
```suggestion
		if(!length(contents))
```

Comment (senorsapo) [reply code/game/objects/items/vaurca.dm]: sir yes sir

**Review by NonQueueingMatt — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**